### ISSUE TITLE
Visualizing the Convolution Layers in TFlearn

### ISSUE THREAD

**Issue by SHAFNehal** [tflearn/tflearn#509]
Is there any way and mixture of ways That I can use to visualize the convolution layers in TFLearn? from the following code I want to print the 'CONV" layer. 

`network = input_data(shape=[None, 227, 227, 3])
network = conv_2d(network, 100, 15, strides=2, activation='relu',padding='same',name='CONV')
network = max_pool_2d(network, 2, strides=2)
network = local_response_normalization(network)
network = fully_connected(network, 25, activation='tanh')
network = regression(network, optimizer='momentum',
                     loss='categorical_crossentropy',
                     learning_rate=0.001)
Nepocs=1000
# Training
model = tflearn.DNN(network, checkpoint_path='model_alexnet',
                    max_checkpoints=1, tensorboard_verbose=2)
model.fit(X, Y, n_epoch=Nepocs, validation_set=0.1, shuffle=True,
          show_metric=True, batch_size=64, snapshot_step=200,
          snapshot_epoch=False, run_id='Conv_Car')`

Thank you in advance

**Comment by aymericdamien** [comment]
What do you mean by visualizing conv layer? Do you want to visualize the weights of a layer or the heatmap given an input image?

**Comment by SHAFNehal** [comment]
I was referring to visualizing the weights of a layers.

**Comment by BBarbosa** [comment]
@aymericdamien I can already visualize the weights. I would also like to see the heatmap given an input image but i didn't manage to do that soo far.
Does the tflearn's convolutional layer output allows to do that? There are many examples for native TensorFlow but i couldn't combine them with tflearn.
With the best regards.

**Comment by germanmarky** [comment]
I also have the same question.
